codegen/projections/white_label/lib/white_label/client.rb

```diff
@@ -137,9 +140,16 @@ def defaults_test(params = {}, options = {}, &block)
         validator: Validators::DefaultsTestInput,
         validate_input: config.validate_input
       )
+      stack.use(Middleware::BeforeMiddleware)
```

Comment:
I don't think we intended for middleware to be relative in runtime, but it was supposed to be relative in code generation?